CONTRIBUTING.md

```diff
@@ -241,6 +241,31 @@ Good docs make developers happy, and we love happy developers! We've got a few d
 * Tutorials/examples
 * Docstrings in Python functions in RST format - generated by Sphinx
 
+### Documentation guidelines
+
+1. Decide if your page is a **guide or a tutorial**. Make sure it fits its section.
+2. Use “**you**” instead of “we” or “I”. It **engages** the reader more.
+3. **Sentence case** for headers. (Use [https://convertcase.net/](https://convertcase.net/) to check)
+4. Keep sentences short. If possible, **fewer than 13 words**.
+5. Only use `backticks` for direct references to code elements.
+6. Jina product names should be capitalized and not backticked (Flow, Executor, Hub etc.).
+7. All **acronyms** should be UPPERCASE (Ex. YAML, JSON, HTTP, SSL).
+8. Think about the **structure** of the page beforehand. Split it into headers before writing the content.
+9. If relevant, include a “**See also**” section at the end.
+10. Link to any existing explanations of the concepts you are using.
+
```

Comment:
```suggestion
9. If relevant, include a “**See also**” section at the end.
10. Link to any existing explanations of the concepts you are using.
Bonus: Know when to break the rules. Documentation writing is as much art as it is science, and sometimes you will have to deviate from these rules in order to write good docs.
```

Reply:
I like it maybe escape between 10 and bonus

Reply:
Edited slightly (rule 4 ;) )